README.md

```diff
@@ -26,9 +26,22 @@
 
 Pode is a Cross-Platform framework for creating web servers to host [REST APIs](https://badgerati.github.io/Pode/Tutorials/Routes/Overview/), [Web Pages](https://badgerati.github.io/Pode/Tutorials/Routes/Examples/WebPages/), and [SMTP/TCP](https://badgerati.github.io/Pode/Servers/) Servers. Pode also allows you to render dynamic files using [`.pode`](https://badgerati.github.io/Pode/Tutorials/Views/Pode/) files, which are just embedded PowerShell, or other [Third-Party](https://badgerati.github.io/Pode/Tutorials/Views/ThirdParty/) template engines. Plus many more features, including [Azure Functions](https://badgerati.github.io/Pode/Hosting/AzureFunctions/) and [AWS Lambda](https://badgerati.github.io/Pode/Hosting/AwsLambda/) support!
 
-<p align="center">
-    <img src="https://github.com/Badgerati/Pode/blob/develop/images/example_code_readme.svg?raw=true" width="70%" />
-</p>
+<div style="display: flex; justify-content: center;">
+    <pre style="background-color: transparent; padding: 10px; border-radius: 5px; border: none;font-size: 18px;">
+
+```powershell
+
+Start-PodeServer -ScriptBlock {
+    Add-PodeEndPoint -Address localhost -port 32005 -Protocol Http
+    Add-PodeRoute -Method Get -Path '/ping' -ScriptBlock {
+        Write-PodeJsonResponse -Value @{value = 'pong' }
+    }
+}
+
+```
+
+</pre>
+</div>
```

Comment:
The styling here won't work in GitHub, as it removes all custom styling. I'm happy for the image to be replaced with just a standard markdown codeblock though :)